src/cephadm/cephadm

```diff
@@ -328,7 +330,7 @@ class SNMPGateway:
             return None
 
         try:
-            with urlopen(f"http://0.0.0.0:{ports[0]}/") as r:
+            with urlopen(f'http://0.0.0.0:{ports[0]}/') as r:
```

Comment:
Did you mean `localhost` instead?. Can you squash this change with the one that added this line?